multitenancy/tenant-manager-api/src/main/java/io/apicurio/multitenant/persistence/dto/RegistryTenantDto.java

```diff
@@ -0,0 +1,88 @@
+package io.apicurio.multitenant.persistence.dto;
+
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import io.apicurio.multitenant.datamodel.RegistryTenant;
+
+@Entity
+public class RegistryTenantDto {
```

Comment:
Since you're using some Panache capabilities, why not extending `PanacheEntity`?

Reply:
I think using `PanacheEntity` couples the app a lot with quarkus dependencies, I think plain JPA is better